packages/analytics-js-integrations/__tests__/integrations/Ninetailed/browser.test.js

```diff
@@ -0,0 +1,95 @@
+import Ninetailed from '../../../src/integrations/Ninetailed/browser';
+
```

Comment:
Add tests for `isLoaded` and `isReady` as well.
Make sure you test the scenario where the integration reports the loaded and ready statuses after a while.